### ISSUE TITLE
Use SVC to predict the final label of embedding data

### ISSUE THREAD

**Issue by holdless** [holdless/Colab#2]
from sklearn.svm import SVC
from sklearn.metrics import accuracy_score

svc = SVC()
svc.fit(train_x_embeddings, all_train_label)

train_prediction = svc.predict(train_x_embeddings)
print("training accuracy : ", accuracy_score(all_train_label, train_prediction))

valid_prediction = svc.predict(valid_x_embeddings)
print("validation accuracy : ", accuracy_score(all_val_label, valid_prediction))
